x11-misc/synergy_haiku/synergy_haiku-0.3.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="Mouse and Keyboard sharing software"
+DESCRIPTION="Synergy brings your computers together in one cohesive experience; its free \
+and open source cross-platform software for sharing one mouse and keyboard \
+between multiple computers on your desk. It works on Windows, Mac OS, Linux \
+and Haiku allowing you to seamlessly move your mouse cursor between computers. \
+
+Credits for: Axel Dörfler (ATKeymap) - Stefano Ceccherini (wrapper) - Jérôme Duval (Keymap) \
+Alex Evans (uSynergy) - Jessica Hamilton (wrapper) - Ed Robbins (wrapper) \
+
+Usage: Create a configuration file at ~/config/settings/synergy with:
+	enable = true
+	server = 192.168.1.X (ip of your server)
+	enableCLipboard = false (false recomended)
+
+For more info and options in config file visit: https://github.com/unarix/synergy_haiku
+"
+HOMEPAGE="https://symless.com/synergy/"
+COPYRIGHT="2020 - Nahuel Tello (Fix & Package) - Axel Dörfler (ATKeymap) - Stefano Ceccherini (wrapper) \
+- Jérôme Duval (Keymap) - Alex Evans (uSynergy) - Jessica Hamilton (wrapper) - Ed Robbins (wrapper)"
```

Comment:
```suggestion
COPYRIGHT="2004-2006 Jérôme Duval
	2005-2010 Axel Dörfler
	2004-2012 Haiku, Inc
	2014 Ed Robbins
	2014 Jessica Hamilton
	2015-2016 Alexander von Gluck IV
	2012 Alex Evans
	2012 Synergy Si Ltd."
```

I'd recommend that you extract the copyrights from each source code file then add them here line-by-line to comply with the MIT license attached to them (the license requires the copyrights to be presented as shown in the code).